tests/Integration/Behaviour/Features/Scenario/Product/add_product.feature

```diff
@@ -0,0 +1,35 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags add
+@reset-database-before-feature
+Feature: Add basic product from Back Office (BO)
+  As a BO user
+  I need to be able to add new product with basic information from the BO
+
+  @add
+  Scenario: I add a product with basic information
+    When I add product "product1" with following basic information:
+      | name | en-US:bottle of beer |
+      | type | standard             |
+    Then product "product1" should have following values:
+      | active           | false          |
+      | condition        | new            |
+    And product "product1" type should be standard
+    And product "product1" "name" should be "en-US:bottle of beer"
+    And product "product1" should be assigned to default category
+
+  @add
+  Scenario: I add a product with invalid type
+    When I add product "product2" with following basic information:
+    | name | en-US:empty box |
+    | type | undefined       |
+  Then I should get error that product type is invalid
+
```

Comment:
Since it's already managed can you add a test about invalid name (with some invalid characters)